helix-rest/src/main/java/org/apache/helix/rest/clusterMaintenanceService/MaintenanceManagementService.java

```diff
@@ -67,10 +67,17 @@
 
 
 public class MaintenanceManagementService {
+  private static class MaintenanceExecutorService {
```

Comment:
It's an overkill to have a class for it. Just define the executor service and instantiate it from java property should be fine.